# $Id$

This file contains patches for lesstif-0.84.  We suggest that these
patches be applied (using `patch -l < lesstif-0.84.patch') before
building LessTif and running DDD with LessTif.

*** lesstif-0.84/lib/Xm/ScrolledW.c.orig	Wed May 27 10:08:20 1998
--- lesstif-0.84/lib/Xm/ScrolledW.c	Wed May 27 11:15:11 1998
***************
*** 53,63 ****
  /* rws 25 Mar 1998
     With this undefined the page list of mgv and mfm work correctly. The mfm
     problem can be seen if you start it and then select a directory from the
     middle list.  At this point the list on the right gets real tiny.
   */
! #undef MFM_BUGS
  
  /* Forward Declarations */
  
  #if 1
  static void class_initialize();
--- 53,63 ----
  /* rws 25 Mar 1998
     With this undefined the page list of mgv and mfm work correctly. The mfm
     problem can be seen if you start it and then select a directory from the
     middle list.  At this point the list on the right gets real tiny.
   */
! #define MFM_BUGS 1
  
  /* Forward Declarations */
  
  #if 1
  static void class_initialize();
***************
*** 2733,2743 ****
  		XmNminimum, &min, 
  		NULL);
  	val -= dy;
  	val = val < min ? min : val;
  	val = val > max ? max : val;
! 	XtVaSetValues((Widget)SW_VSB(scrollw_widget), XmNvalue, dy, NULL);
      }
  }
  
  void
  XmScrolledWindowSetAreas(Widget widget,
--- 2733,2743 ----
  		XmNminimum, &min, 
  		NULL);
  	val -= dy;
  	val = val < min ? min : val;
  	val = val > max ? max : val;
! 	XtVaSetValues((Widget)SW_VSB(scrollw_widget), XmNvalue, val, NULL);
      }
  }
  
  void
  XmScrolledWindowSetAreas(Widget widget,
